docs/pages/button.js

```diff
@@ -187,46 +169,34 @@ card(
         required: false,
         defaultValue: 0,
         description: [
-          'Remove the component from sequential keyboard navigation to improve accessibility. The component is not focusable with keyboard navigation but it can be focused with Javascript or visually by clicking with the mouse.',
-          `The default behaviour for the component is to be focusable in sequential keyboard navigation in the order defined by the document's source order.`,
-          `If component is disabled, the component is also unreachable from keyboard navigation.`,
+          'Use "-1" to remove Button from keyboard navigation. See the [Accessibility guidelines](/accessibility) to learn more.',
         ],
-        href: 'type-roles',
       },
       {
         name: 'role',
         type: `'button' | 'link'`,
         required: false,
         defaultValue: 'button',
-        description: [
-          `Select a button variant:`,
-          `- 'button': Use to render 'submit' or 'button'-type buttons. The button is rendered as a \`<button>\`.`,
-          `- 'link': Use for buttons to act like links. The button is rendered as an \`<a>\`.`,
-          `Required with link-role buttons.`,
-        ],
-        href: 'type-roles',
+        description: ['Use “link” to indicate Button that is acting as an `<a>` link.'],
```

Comment:
i'd create a variant for roles. If OOO, I'd open a JIra ticket to mirror IconButton

Reply:
Creating a ticket!